org.osgi.test.cases.cm/src/org/osgi/test/cases/cm/junit/ConfigurationPluginTests.java

```diff
@@ -87,21 +115,25 @@ protected void setUp() throws Exception {
         }
 	}
 
+	private long getLongProperty(String string, int i) {
```

Comment:
What is this used for?

Reply:
The original test hat optional to set the sleep time.. this is just something i will clean up when tests work every time